test/parallel/test-https-unix-socket-self-signed.js

```diff
@@ -9,8 +9,8 @@ common.refreshTmpDir();
 const fs = require('fs');
 const https = require('https');
 const options = {
-  cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
-  key: fs.readFileSync(common.fixturesDir + '/test_key.pem')
+  cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`),
```

Comment:
This is Ok.
But it would be much better to use `path.join` instead, see https://github.com/nodejs/node/pull/14272#discussion_r127608804